sct-commons/src/main/java/org/lfenergy/compas/sct/commons/dto/LDEPFSettingsSupplier.java

```diff
@@ -79,18 +78,31 @@ record LDEPFSetting(String rteIedType, String iedRedundancy, BigInteger iedInsta
                             .orElse(true));
 
     BiPredicate<AbstractLNAdapter<?>, LDEPFSetting> isValidDataTypeTemplate = (lnAdapter, setting) -> {
-       String doName = setting.doInst() == null || setting.doInst().equals("0") ? setting.doName() : setting.doName()+setting.doInst();
-       var lnType = lnAdapter.getLnType();
-        return StringUtils.isEmpty(doName) || StringUtils.isEmpty(setting.daName()) ||
-                lnAdapter.getDataTypeTemplateAdapter().getLNodeTypeAdapterById(lnType)
-                .filter(lNodeTypeAdapter -> {
-                    try {
-                        lNodeTypeAdapter.check(new DoTypeName(doName), new DaTypeName(setting.daName()));
-                    } catch (ScdException ex) {
-                        return false;
-                    }
-                    return true;
-                }).isPresent();
+        if(StringUtils.isEmpty(setting.doName())){
+            return true;
+        }
+        String doName = setting.doInst() == null || setting.doInst().equals("0") ? setting.doName() : setting.doName()+setting.doInst();
```

Comment:
Can you format the file using IntelliJ default format ?
This should add spaces around the `+` sign for example.

Reply:
done